src/test/component.test.js

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import '@testing-library/jest-dom/extend-expect';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Home from '../components/StockList';
+import Nav from '../components/Nav';
+import store from '../redux/configureStore';
+
```

Comment:
- [ ] A test case is failing. Kindly ensure to improve upon the same.
![image](https://user-images.githubusercontent.com/12745474/195991572-853c8388-adae-4dbf-a416-5a2eac68d9b7.png)